instance/models/mixins/ansible.py

```diff
@@ -140,29 +140,14 @@ def deploy(self):
         """
         Deploy instance to the active servers
         """
-        for attempt in range(self.attempts):
-            with open_repository(self.ansible_source_repo_url,
-                                 ref=self.configuration_version) as configuration_repo:
-                playbook_path = os.path.join(configuration_repo.working_dir,
-                                             'playbooks')
-                requirements_path = os.path.join(configuration_repo.working_dir,
-                                                 'requirements.txt')
+        with open_repository(self.ansible_source_repo_url, ref=self.configuration_version) as configuration_repo:
+            playbook_path = os.path.join(configuration_repo.working_dir, 'playbooks')
+            requirements_path = os.path.join(configuration_repo.working_dir, 'requirements.txt')
 
-                log = ('Running playbook "{path}/{name}" attempt {attempt} of '
-                       '{attempts}:').format(
-                           path=playbook_path,
-                           name=self.ansible_playbook_name,
-                           attempts=self.attempts,
-                           attempt=attempt + 1)
-                self.logger.info(log)
-                log, returncode = self._run_playbook(requirements_path, playbook_path)
-                if returncode != 0:
-                    self.logger.error(
-                        'Playbook failed for instance {}'.format(self))
-                    continue
-                else:
-                    break
+            log = 'Running playbook "{path}/{name}":'.format(path=playbook_path, name=self.ansible_playbook_name)
+            self.logger.info(log)
```

Comment:
We don't need to store the string in a variable first, in particular given that the same variable name is used for something else in the next line.  (I know you didn't introduce this, but we might just as well fix it while we are here.)